src/main/java/org/openstreetmap/atlas/geography/atlas/raw/creation/OsmPbfReader.java

```diff
@@ -252,7 +278,7 @@ private RelationBean constructRelationBean(final Relation relation)
                 }
                 else
                 {
-                    logger.warn(MISSING_MEMBER_MESSAGE, relation.getId(), EntityType.Node,
+                    logger.debug(MISSING_MEMBER_MESSAGE, relation.getId(), EntityType.Node,
```

Comment:
`trace` even, no? This happens a lot because of the way osmisis cuts pbfs based on a shard boundary...